### ISSUE TITLE
rc core/command results in a 500 Internal Server Error

### ISSUE THREAD

**Issue by siopaonow** [rclone/rclone#4914]
#### What is the problem you are having with rclone?
While using the `rc core/command` command to run a rclone terminal command remotly results in a 500 Internal Server Error. Using other commands like `rc rc/noop` work as expected.
Documentation: https://rclone.org/rc/#core-command
Related forum post: https://forum.rclone.org/t/error-running-remote-control/20435/5

#### What is your rclone version (output from `rclone version`)
```
rclone v1.54.0-beta.5050.ef2bfb971
- os/arch: linux/amd64
- go version: go1.15.6
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
OS: Windows 10, 64 bit
WSL: Version 2, Ubuntu

####  Which cloud storage system are you using? (e.g. Google Drive)
N/A

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
`rclone -vv rcd --rc-no-auth` && `rclone -vv rc core/command command=version`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
```
$: rclone -vv rcd --rc-no-auth
2021/01/09 16:35:09 DEBUG : rclone: Version "v1.54.0-beta.5050.ef2bfb971" starting with parameters ["rclone" "-vv" "rcd" "--rc-no-auth"]
2021/01/09 16:35:09 NOTICE: Serving remote control on http://localhost:5572/
2021/01/09 16:35:16 DEBUG : rc: "core/command": with parameters map[_request:0xc00061a900 _response:0xc00022e000 command:version]
2021/01/09 16:35:16 ERROR : rc: "core/command": error: response object is required
expecting *http.ResponseWriter value for key "_response" (was *http.response)
2021/01/09 16:35:16 ERROR : rc: failed to write JSON output: json: unsupported type: func() (io.ReadCloser, error)
```
```
$: rclone -vv rc core/command command=version
2021/01/09 16:36:02 DEBUG : rclone: Version "v1.54.0-beta.5050.ef2bfb971" starting with parameters ["rclone" "-vv" "rc" "core/command" "command=version"]
2021/01/09 16:36:02 DEBUG : 4 go routines active
2021/01/09 16:36:02 Failed to rc: Failed to read rc response: 500 Internal Server Error:
```
Using the API directly results in the same error, for example: `curl --data "command=version" http://localhost:5572/core/command`

**Comment by ncw** [comment]
I've reproduced this problem.

It shows an egregious lack of testing of this bit of code alas!

Can you try this

[v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command](https://beta.rclone.org/branch/fix-4914-rc-command/v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command/) on branch [fix-4914-rc-command](https://github.com/rclone/rclone/tree/fix-4914-rc-command) (uploaded in 15-30 mins)

I need to write some tests for that too :-)

**Comment by siopaonow** [comment]
Looks like the above build failed for the linux binaries, but I tested the freebsd ones and I can confirm the command it working as expected:
```
$: rclone -vv rcd --rc-no-auth
2021/01/10 12:14:39 DEBUG : rclone: Version "v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command" starting with parameters ["./rclone" "-vv" "rcd" "--rc-no-auth"]
2021/01/10 12:14:39 NOTICE: Serving remote control on http://localhost:5572/
2021/01/10 12:14:41 DEBUG : rc: "core/command": with parameters map[_request:0xc0007f2100 _response:0xc00001e000 command:version]
2021/01/10 12:14:41 DEBUG : rc: "core/command": reply map[error:false result:rclone v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command
- os/arch: freebsd/amd64
- go version: go1.15.6
]: <nil>
```
```
$: rclone -vv rc core/command command=version
2021/01/10 12:14:41 DEBUG : rclone: Version "v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command" starting with parameters ["./rclone" "-vv" "rc" "core/command" "command=version"]
{
        "error": false,
        "result": "rclone v1.54.0-beta.5050.e3efbcc69.fix-4914-rc-command\n- os/arch: freebsd/amd64\n- go version: go1.15.6\n"
}
2021/01/10 12:14:41 DEBUG : 4 go routines active
```

**Comment by ncw** [comment]
Thanks for testing.

Here is the next version [v1.54.0-beta.5052.a73addb71.fix-4914-rc-command](https://beta.rclone.org/branch/fix-4914-rc-command/v1.54.0-beta.5052.a73addb71.fix-4914-rc-command/) on branch [fix-4914-rc-command](https://github.com/rclone/rclone/tree/fix-4914-rc-command) (uploaded in 15-30 mins)

This now has tests and a few more fixes! It should build linux binaries this time :crossed_fingers:

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.54
